### PR TITLE
Update document.

### PR DESCRIPTION
**Pull request by lipingxue**
**What this PR does / why we need it**:
Update document  self-signed-certificate.md to explain how self-signed certicated passed in during installation is used.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```NONE

```

### CODE REVIEW

File: docs/self-signed-certificate.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ To keep consistency with velero, Velero plugin for vsphere also supports using v
 * Guest cluster: supported since v1.1.1
 * Supervisor cluster: not supported
 
-To install with a storage provider secured by a self-signed certificate, the --cacert option needs to be added to provide a path to a certificate bundle to trust.
+To install with a storage provider secured by a self-signed certificate, the --cacert option needs to be added to provide a path to a certificate bundle to trust. This certificate is used to verify TLS connection to the storage provider when backing up and restoring.
```
Comment (xing-yang) [inline docs/self-signed-certificate.md:11]: Please clarify that this is for the Backup Storage Location such as an object store.

Comment (xing-yang) [reply docs/self-signed-certificate.md:11]: Can you also add a note at the top doc level where this is referenced?

Comment (lipingxue) [reply docs/self-signed-certificate.md:11]: @xing-yang  This doc is referenced by guest.md, supervisor.md and vanilla.md. What should we put in the note?

**Review by xing-yang — APPROVED** [review]
LGTM

**Pull request merged**